core/node/state_keeper/src/batch_executor/main_executor.rs

```diff
@@ -207,6 +210,9 @@ impl CommandReceiver {
         // There is some post-processing work that the VM needs to do before the block is fully processed.
         let result = vm.finish_batch();
         if result.block_tip_execution_result.result.is_failed() {
+            KEEPER_METRICS.tx_execution_result
+                [&metrics::TxExecutionResult::from(&result.block_tip_execution_result.result)]
+                .inc();
```

Comment:
I think we shouldn't increment metric here as batch tip is not a transaction and it's "successful" in 100% of cases (if it's not then server panics in the next line)